src/main/resources/GoogleApps/Translations.fr.xml

```diff
@@ -39,17 +39,27 @@
   <minorEdit>false</minorEdit>
   <syntaxId>plain/1.0</syntaxId>
   <hidden>false</hidden>
-  <content>admin.googleapps=Google Apps
+  <content><![CDATA[
+admin.googleapps=Google Apps
 googleapps.config.heading=Configuration Intégration Google Apps
-googleapps.config.explanation=XWiki peut-etre intégré avec Google Apps, ce qui permet ensuite l"édition des attachements Office en utilisant les éditeurs Google Apps ainsi que leur sauvegarde dans le wiki. Cela permet également de télécharger un fichier Google Apps dans n"importe quelle page du Wiki. Afin d'activer Google Apps vous devez vous [[enregistrer pour l'accès à OAuth sur Google.com&gt;&gt;https://accounts.google.com/ManageDomains]] et insérer votre nom de domaine et la clé dans les paramètres suivants.
+googleapps.config.explanation=XWiki peut-etre intégré avec Google Apps, ce qui permet ensuite l'édition des attachements Office en utilisant les éditeurs Google Apps ainsi que leur sauvegarde dans le wiki. Cela permet également de télécharger un fichier Google Apps dans n'importe quelle page du Wiki. Afin d'activer Google Apps vous devez vous [[enregistrer pour l'accès à OAuth sur Google.com>>https://accounts.google.com/ManageDomains]] et insérer les informations dans les paramètres suivants (voir aussi  please see [[les instructions d'installation>>https://store.xwiki.com/xwiki/bin/view/Extension/GoogleAppsIntegration#installation]]).
```

Comment:
`voir aussi  please see` should be fixed here and also to use parametrized translation